lib/src/over_react_test/react_util.dart

```diff
@@ -422,6 +423,10 @@ Element getComponentRootDomByTestId(dynamic root, String value, {String key = de
 /// Returns the [Element] of the first descendant of [root] that has its [key] html attribute value set to a
 /// space-delimited string containing [value].
 ///
+/// Setting [searchInShadowDom] to true will allow the query to search within ShadowRoots that use `mode:"open"`.
+/// [shadowDepth] will limit how many layers of ShadowDOM will searched, by default it will seach infinately deep, this
+/// should only be needed if there is alot of ShadowRoots within ShadowRoots.
```

Comment:
Comma splice / typos.
```suggestion
/// [shadowDepth] will limit how many layers of ShadowDOM will searched. By default it will search infinitely deep, and this
/// should only be needed if there are a lot of ShadowRoots within ShadowRoots.
```